idprovider/local/cache.go

```diff
@@ -81,17 +81,17 @@ func (c *contactsCache) ReadByOffChainAddr(offChainAddr pwire.Address) (_ perun.
 	return c.readByAlias(alias)
 }
 
-// Write adds the peer to contacts cache. Returns an error if the alias is already used by same or different peer or,
-// if the off-chain address string of the peer cannot be parsed using the wallet backend of this contacts provider.
-func (c *contactsCache) Write(alias string, p perun.Peer) error {
+// Write adds the peer to idProvider cache. Returns an error if the alias is already used by same or different peer or,
+// if the off-chain address string of the peer cannot be parsed using the wallet backend of this idProvider.
+func (c *idProviderCache) Write(alias string, p perun.Peer) error {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
 
 	if oldPeer, ok := c.peersByAlias[alias]; ok {
 		if PeerEqual(oldPeer, p) {
-			return errors.New("peer already present in contacts")
+			return errors.New("peer already present in idProvider")
 		}
-		return errors.New("alias already used by another peer in contacts")
+		return errors.New("alias already used by another peer in idProvider")
```

Comment:
```suggestion
		return errors.New("alias already used by another peer in ID provider")
```

same as above 
Using ID provider as two words is more readable in the context of comments and documentation.